manifests/package/debian.pp

```diff
@@ -32,6 +32,10 @@
     include '::apt'
     Exec['apt_update'] -> Package['nginx']
 
+    ensure_packages([ 'apt-transport-https', 'ca-certificates' ])
```

Comment:
And yes, you're right, I didn't catch that this was inside
```
  if $manage_repo {
```
already, so this should be fine... it just wasn't clear from context. 
However, there still should probably be some tests in `classes/nginx_spec.rb`
in line 191, where there are tests for `manage_repo=false`, you can add a test for the absence of the package, e.g.,
```
        it { is_expected.not_to contain_package('apt-transport-https') }
```
and in the section starting at line 155, an equivalent test for its presence.